iamctl/pkg/claims/import.go

```diff
@@ -0,0 +1,165 @@
+/**
+* Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+*
+* WSO2 LLC. licenses this file to you under the Apache License,
+* Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package claims
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/wso2-extensions/identity-tools-cli/iamctl/pkg/utils"
+	"gopkg.in/yaml.v2"
+)
+
+func ImportAll(inputDirPath string) {
+
+	log.Println("Importing Claim Dialects...")
+	importFilePath := filepath.Join(inputDirPath, utils.CLAIMS)
+
+	var files []os.FileInfo
+	if _, err := os.Stat(importFilePath); os.IsNotExist(err) {
+		log.Println("No claim dialects to import.")
+	} else {
+		files, err = ioutil.ReadDir(importFilePath)
+		if err != nil {
+			log.Println("Error importing claim dialects: ", err)
+		}
+		if utils.TOOL_CONFIGS.AllowDelete {
+			removeDeletedDeployedClaimdialect(files, importFilePath)
+		}
+	}
+
+	for i, file := range files {
```

Comment:
Let's add a comment explaining the logic here

Reply:
Added a comment.